docs/the_black_code_style/future_style.md

```diff
@@ -1,117 +1,38 @@
 # The (future of the) Black code style
 
-```{warning}
-Changes to this document often aren't tied and don't relate to releases of
-_Black_. It's recommended that you read the latest version available.
-```
-
-## Using backslashes for with statements
```

Comment:
Moved this to the end as it's a speculative feature that we haven't even implemented. The real-world preview style is more important.